duckduckgo/iOS
# disable login detection

Task/Issue URL: https://app.asana.com/0/392891325557410/1164681539353125
Tech Design URL:
CC:

**Description**:

Emergency fix to lock down login detection.  Existing code is still mostly in place, just disabled for now.  Will tidy up/remove/fix as part of follow up project.

**Steps to test this PR**:

1. Migrate from previous version - ensure that previously detected domains and cookies are removed (won't affect other sites that are logged in but not detected)
1. Fireproof a website using the menu item - should stay logged in after using the fire button / auto-clear
1. Ensure settings still allows remove / remove all of domains

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
